config/initializers/cloudinary.rb

```diff
@@ -0,0 +1,7 @@
+Cloudinary.config do |config|
+  config.cloud_name = Rails.application.credentials.cloudinary.fetch(:cloud_name)
+  config.api_key = Rails.application.credentials.cloudinary.fetch(:api_key)
+  config.api_secret = Rails.application.credentials.cloudinary.fetch(:api_secret)
+  config.secure = true
+  config.cdn_subdomain = true
+end
```

Comment:
You can make a little helper method for accessing deeply into `Rails.application.credentials.cloudinary` 3 times. This technique is called memoizing and it will reduce a tiny tiny bit of time it takes to access these settings (less important here) and it will show you one of the syntaxes for memoizing (more important here) and dry up this code bit (good for making things more readable).

```suggestion
Cloudinary.config do |config|
  config.cloud_name = cloudinary_credentials(:cloud_name)
  config.api_key = cloudinary_credentials(:api_key)
  config.api_secret = cloudinary_credentials(:api_secret)
  config.secure = true
  config.cdn_subdomain = true
end

# Memoizes and accesses credentials stored at Rails.application.credentials.cloudinary
# @param [Symbol] key
# @return [String]
def cloudinary_credentials(key)
  @cloudinary_credentials ||= Rails.application.credentials.cloudinary
  @cloudinary_credentials.fetch(key.to_sym)
end
```